static/app/views/monitors/monitorIssues.tsx

```diff
@@ -8,19 +10,34 @@ type Props = {
   orgId: string;
 };
 
-const MonitorIssues = ({orgId, monitor}: Props) => (
-  <IssueList
-    endpoint={`/organizations/${orgId}/issues/`}
-    query={{
-      query: 'monitor.id:"' + monitor.id + '"',
-      project: monitor.project.id,
-      limit: 5,
-    }}
-    pagination={false}
-    emptyText={t('No issues found')}
-    noBorder
-    noMargin
-  />
-);
+const MonitorIssues = ({orgId, monitor}: Props) => {
+  const renderEmptyMessage = () => (
```

Comment:
nit: shall we move this out of the component? maybe sometime soon we could maybe it also reusable since we are using a similar code in many different places in the UI